src/Components/EditProduct/Form.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import EditForm from './EditForm'
+import { Card, Col, Row, Button} from 'react-bootstrap'
+
+function Form(props)
+{
+	return(
+		<div>
+			<Card>
+				<Card.Title className='text-center'>
+					<Col>
+						<Row>
+		                    <Col sm={{span:2}} style={{fontSize:"2em"}}>
+		                    	<Button variant="outline-dark">
+									<i className="fas fa-chevron-left align-center"></i>
+								</Button>
+		                    </Col>
+		                    <Col sm={{span:8}}>
+		                        Buyer's regret
+		                    </Col>
+		                     <Col sm={{span:2}} style={{fontSize:"2em"}}>
+		                     	<i className="far fa-user-circle align-middle"></i>
+		                     </Col> 
+		                </Row>
+		                <Row>
+		                <Col sm={{span:2}}></Col>
+		                <Col sm={{span:8}}>
+		                        Product Details
+		                </Col>
+		                <Col sm={{span:2}}></Col>
+		                </Row>
+		            </Col>

```

Comment:
This card with title, lines 10 to 34, is going to be a separate component, i guess, but it is beyond  the scope of this task and it is fine to keep it copy-pasted for now.